server/metrics/fdb.go

```diff
@@ -38,23 +42,42 @@ var (
 		"Update",
 		"UpdateRange",
 		"BeginTx",
-		"GetInternalDatabase",
 	}
 )
 
-func GetFdbReqTags(reqMethodName string, tx bool) map[string]string {
+func getFdbReqTags(reqMethodName string, tx bool) map[string]string {
 	return map[string]string{
-		"method": reqMethodName,
-		"tx":     strconv.FormatBool(tx),
+		"method":        reqMethodName,
+		"tx":            strconv.FormatBool(tx),
```

Comment:
everything is transactional, do we need this extra tag for now? Or pass it in the method for completion but no need to emit it?